src/file-utils/edfi_lms_file_utils/file_reader.py

```diff
@@ -18,17 +18,42 @@ def _default() -> pd.DataFrame:
     return pd.DataFrame()
 
 
-def _read_csv(file: str, nrows: Union[int, None] = None) -> pd.DataFrame:
+def _read_csv(
+    file: str,
+    nrows: Optional[int] = None,
+    data_types: Dict[str, str] = dict(),
+    date_cols: List[str] = list(),
```

Comment:
```suggestion
    specific_date_cols: List[str] = list(),
```
It'd be helpful to somehow specify that there're some common columns already defined... It cold be renaming the parameter, or adding docstrings(even when it is a privated function)

Reply:
Renaming to `extra_date_columns` seems appropriate, along with a docstring. I'll make that change.